README.MD

```diff
@@ -0,0 +1,128 @@
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Book] <a name="about-project"></a>
+
+This project is about Creating a collection of book list.
+In this project, we have created a website that allows the
+users to add and remove books from book list.
+This will be done using javascript objects and arrays.
+
+## 🚀 VISIT SITE <a name="website"></a>
+
+<a name="https://ritadee.github.io/pair-programming-awesomebook/>">https://ritadee.github.io/pair-programming-awesomebook/</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML, CSS AND JAVASCRIPT  
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS & JS</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- JavaScript file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```git@github.com:RitaDee/Awesome-books-pair.git```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+1. Right click index.html file and run in browser.
+2. Open index file in vscode right click and open in live server
+```
+
+## 👥 Author <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Rita Daniel**
+
+- GitHub: [@RitaDee](https://github.com/RitaDee)
+- Twitter: [@durdana_dee](https://twitter.com/durdana_dee)
+- LinkedIn: [Rita Daniel](https://www.linkedin.com/in/rita-daniel/)
+
+👤 **Ismail Munyentwari**
+
+- GitHub: [@Ismail](https://github.com/ismailmunyentwari9)
+- Instagram: [@ismail_0258](https://www.instagram.com/ismail_0258/)
+- LinkedIn: [ismail Munyentwari](https://www.linkedin.com/in/munyentwari-ismail-754718191/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+```I would like to thank the Microverse learning resources for all the helpful information rendered in completion of this project.
+
+```I give credit to ourselves for collaborating on this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] it would be better to correct the License section and make sure to add MIT file and the MIT link should be working and it takes to the license